code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -394,21 +394,59 @@
 
 /datum/reagent/medicine/calomel
 	name = "Calomel"
-	description = "Quickly purges the body of toxic chemicals. Toxin damage is dealt if the patient is in good condition."
+	description = "Quickly purges the body of all chemicals except itself. The more health a person has, the more toxin damage it will deal"
```

Comment:
needs a period and needs to mention that it can heal the tox damage of people who are at low enough health